src/core/ledger/ledger.js

```diff
@@ -575,6 +602,58 @@ export class Ledger {
     account.identity = updatedIdentity;
   }
 
+  /**
+   * setPayoutAddress allows participants to set a payable address to collect
+   * grain. These addresses are keyed on a specific currency, which ensures that
+   * users don't erroneously receive a grain distribution to an address that
+   * cannot handle it (such as a custodial wallet, or rigidly-designed
+   * contract) and effectively lose that reward.
+   *
+   * An address may be deleted by passing in an empty string for the
+   * `address` parameter. This is useful in case the underlying private key
```

Comment:
you mention an `address` param but there isn't one, should be `payoutAddress`. Also, instead of an empty string should it be passing null since that's a valid type for that param?